cluster/juju/layers/kubernetes-e2e/actions/test

```diff
@@ -16,7 +17,6 @@ fi
 # get the host from the config file
 SERVER=$(cat /home/ubuntu/.kube/config | grep server | sed 's/    server://')
 
-# Execute the E2E test
-e2e.test -kubeconfig /home/ubuntu/.kube/config -host $SERVER -ginkgo.focus $FOCUS -ginkgo.skip $SKIP 2>&1 | tee /home/ubuntu/${JUJU_ACTION_UUID}.log
+ginkgo -nodes=$PARALLELISM $(which e2e.test) -- -kubeconfig /home/ubuntu/.kube/config -host $SERVER -ginkgo.focus $FOCUS -ginkgo.skip "$SKIP|\[Serial\]" 2>&1 | tee /home/ubuntu/${JUJU_ACTION_UUID}.log
```

Comment:
I don't like the idea of force skipping serial, there's not effective way to E2E test for only serial which we still would want to do. Could we instead add `\[Serial\]` to default skip?